app/routers/dogfinder.py

```diff
@@ -26,6 +27,7 @@
 
 IS_FOUND_FIELD = "isFound"
```

Comment:
If it is alright with you, can you please use underscore notation instead of camel case?
"is_found", "dog_id"

Reply:
I wanted to use camelCase since these are values that go into the json api, which works with camel. I can change it to use snake_case and do a transformation logic in the method if you think that is better

Reply:
What do you mean these are values that go into the json api?
json values has issue with snake case notation?

Reply:
as far as I'm familier json convention uses camel case, and this is what will probably be expected in the front

Reply:
I see, well ok. I don't think it really is an issue for the FE but sure lets go with this.